web/src/compose.js

```diff
@@ -9,6 +9,7 @@ import render_wildcard_mention_not_allowed_error from "../templates/compose_bann
 import * as channel from "./channel";
```

Comment:
I'm just leaving this comment here on the first file but please see https://chat.zulip.org/#narrow/stream/137-feedback/topic/Disabling.20show.20more/near/1836505 for a suggestion from @alya:

"I'm thinking we should just mention this when documenting Make it possible to automatically split long messages [#28626](https://github.com/zulip/zulip/pull/28626). That will put the information in the context of where it's actionable."

The thing to mention (as I understand it from the topic above) is that long message are split after they reach 65% of the screen height.

Reply:
The linked CZO thread doesn't seem to mention splitting messages after 65% height.
@alya Do we wish to do that?

Reply:
Oh, sorry if I was unclear. I was not proposing automatically splitting messages at 65% screen height (or at all).